pkgs/top-level/perl-packages.nix

```diff
@@ -14876,6 +14876,7 @@ let
     };
     propagatedBuildInputs = [ IOAsync Moo NetFrameLayerIPv6 namespaceclean ];
     buildInputs = [ TestFatal ];
+    preCheck = "rm -f t/icmp_ps.t t/icmpv6_ps.t"; # ping socket tests fail
```

Comment:
```suggestion
    preCheck = "rm t/icmp_ps.t t/icmpv6_ps.t"; # ping socket tests fail
```
Is -f needed?

Reply:
Probably not, we can drop it :+1:  thanks!

Reply:
This is not a nitpick. rm -f should be avoided all the time or we don't notice if the file vanishes.